### ISSUE TITLE
Lipid kinase activity (GO:0001727) children

### ISSUE THREAD

**Issue by sjm41** [geneontology/go-ontology#18015]
Currently "lipid kinase activity" (GO:0001727) only has two is_a children:
ceramide kinase activity (GO:0001729)
sphinganine kinase activity (GO:0008481)

I was expecting to find several other kinases classified as "lipid kinases", including:

- [x] diacylglycerol kinase activity (GO:0004143) changed to child of lipid kinase activity 

- [x] D-erythro-sphingosine kinase activity (GO:0017050) changed to child of lipid kinase activity 

and the several phosphatidylinositol kinase terms:
- [x] phosphatidylinositol kinase activity (GO:0052742) changed to child of lipid kinase activity 
- [x] phosphatidylinositol phosphate kinase activity (GO:0016307) changed to child of lipid kinase activity
- [x] phosphatidylinositol bisphosphate kinase activity (GO:0052813) changed to child of lipid kinase activity
- [x] phosphatidylinositol 3-kinase activity (GO:0035004) changed to child of lipid kinase activity

(and there may be others...)

All of these other terms are currently direct children of "kinase activity" rather than subtypes of "lipid kinase activity":
![Screen Shot 2019-10-17 at 13 41 10](https://user-images.githubusercontent.com/16306861/67009835-60473580-f0e4-11e9-806a-ceefd57ce61a.png)

Maybe this is just a mistake/oversight, or is it a matter of whether these substrates are classed as lipids or not?
Notably, the children under "lipid binding" (GO:0008289) or "lipid phosphorylation" (GO:0046834) DO appear to classify these molecules as lipids (see image2 and image3), so there's at least some inconsistency here.
![Screen Shot 2019-10-17 at 13 34 50](https://user-images.githubusercontent.com/16306861/67009862-7228d880-f0e4-11e9-8057-f746297c4a96.png)
![Screen Shot 2019-10-17 at 13 37 39](https://user-images.githubusercontent.com/16306861/67009868-7654f600-f0e4-11e9-8d6f-0ae631fc0b53.png)

**Comment by sjm41** [comment]
@hattrill

**Comment by ukemi** [comment]
This classification would be based on the substrate and will be dependent on ChEBI. This classification should happen automatically once we have adopted the Rhea definitions for these terms. @hdrabkin can you investigate the xref situation? This might be a really good example.

**Comment by sjm41** [comment]
Hi David/Harold, Just wondering if we have a rough timescale for when this might happen:
"_This classification would be based on the substrate and will be dependent on ChEBI. This classification should happen automatically once we have adopted the Rhea definitions for these terms_."
Cheers.

**Comment by sjm41** [comment]
Following up David's previous suggestion to "_investigate the xref situation_", here are the Rhea xrefs on the terms (or their children) mentioned in my original post in the current GO:

ceramide kinase activity (GO:0001729) -> xref: RHEA:17929

sphinganine kinase activity (GO:0008481) -> xref: RHEA:15465

diacylglycerol kinase activity (GO:0004143) -> xref: RHEA:10272

D-erythro-sphingosine kinase activity (GO:0017050) -> no RHEA

phosphatidylinositol kinase activity (GO:0052742) -> no RHEA (expected - grouping term)
1-phosphatidylinositol-3-kinase activity (GO:0016303) -> xref: RHEA:12709
1-phosphatidylinositol 4-kinase activity (GO:0004430) -> xref: RHEA:19877
1-phosphatidylinositol-5-kinase activity (GO:0052810) -> no RHEA

phosphatidylinositol phosphate kinase activity (GO:0016307) -> no RHEA (expected - grouping term)
1-phosphatidylinositol-3-phosphate 4-kinase activity (GO:0052811) -> no RHEA
1-phosphatidylinositol-3-phosphate 5-kinase activity (GO:0000285) -> xref: RHEA:13609
1-phosphatidylinositol-4-phosphate 3-kinase activity (GO:0035005) -> xref: RHEA:18373
1-phosphatidylinositol-4-phosphate 5-kinase activity (GO:0016308) -> xref: RHEA:14425
1-phosphatidylinositol-5-phosphate 4-kinase activity (GO:0016309) -> xref: RHEA:12280

phosphatidylinositol bisphosphate kinase activity (GO:0052813) -> no RHEA (expected - grouping term)
phosphatidylinositol-3,4-bisphosphate 5-kinase activity (GO:0052812) -> no RHEA
phosphatidylinositol-4,5-bisphosphate 3-kinase activity (GO:0046934) -> xref: RHEA:21292

phosphatidylinositol 3-kinase activity (GO:0035004) -> xref: RHEA:12709 (should this be on the grouping term?)
1-phosphatidylinositol-3-kinase activity (GO:0016303) -> xref: RHEA:12709
1-phosphatidylinositol-4-phosphate 3-kinase activity (GO:0035005) -> xref: RHEA:18373

**Comment by hdrabkin** [comment]
RHEA has given me mapping  tables and I am going through them; 
RHEA DOES have a mapping of  GO:0035004) -> xref: RHEA:12709;  They have cross referrenced EC:2.7.1.137; ATP + 1-phosphatidyl-1D-myo-inositol <=> ADP + 1-phosphatidyl-1D-myo-inositol 3-phosphate; but there is a problem
The CHEBI ids for the substrates and products  by REHA maps to 1-phosphatidyl-1D-myo-inositol(1−), and 1-phosphatidyl-1D-myo-inositol 3-phosphate(3−) respectively, which are used in the EC def. 
The GO def appears out of sync with the other mappings. here (metacyc, etcs) 
This is also the only term that is mapped in GO to this EC in protege, but I see that it is also used in one of the children. 

I see that the 35004 term has a RHEA but so does the 16303 term. 
The RHEA can only be in one place; it needs to be removed from the 35004 term   (which IS the actual reaction AND is an alternate name in EC:2.7.1.137;
Basically, this grouping term should be stripped of the EC AND RHEA; Note that it is GO that imposes the hierarchy/grouping. THE EC actually  belongs with 35004.

**Comment by hdrabkin** [comment]
THe only reason it is in the mapping files from RHEA is because it had the EC; and now I notice that GO:0016303 is ALSO mapped to the same RHEA in the mapping files (so I need to get rid of the ec and rhea in the grouping term for sure

**Comment by hdrabkin** [comment]
This is all a side issue to this ticket; sorry; but Since I see this problem I should fix it while I see it.)

**Comment by hdrabkin** [comment]
EC and RHEA mappings to GO:0035004; left on GO:0016303

**Comment by sjm41** [comment]
Thanks for sorting out that issue with xrefs on the GO:0035004 and GO:0016303 terms.

Returning to the main issue and David's previous comment: once you've gone through the Rhea mapping tables for these terms, and "adopted the Rhea definitions", then we might be able to rationalise the GO terms coming under lipid kinase activity?

**Comment by hdrabkin** [comment]
One thing is we need to be careful with assigning mappings to grouping terms!
And we need to put a check into Travis RHEA in more than one GO term!

**Comment by ValWood** [comment]
This might be done?

**Comment by hdrabkin** [comment]
the check will be put in AFTER we are all sorted out with dual mappings. I have a merges to do not related to this ticket.

**Comment by sjm41** [comment]
Yes, definitely not done and ticket should stay open.  ("lipid kinase activity" (GO:0001727) still only has two is_a children in the GO.)

**Comment by sjm41** [comment]
Excellent, thanks!  Closing.
